### PR TITLE
Created recommended button

### PR DESCRIPTION
**Pull request by OkechukwuNnujiJohn**

### CODE REVIEW

**Comment by OkechukwuNnujiJohn** [comment]
In the previous commit (First-Technical-challenge-Recommneded-page), I created the filtered display with my algorithm.
In the process I want to: 
--Display only the top 5-8 items in the recommended page
-- Store the recommendation data into the database

https://www.loom.com/share/095c96fbedca4338a81411e9c0f9a442?sid=8989aa0c-b42e-461a-b007-61fce85bbe4a

**Comment by OkechukwuNnujiJohn** [comment]
In the "Uploaded-garment\and\fetched-garment" I started working on the try-on feature/page. Now the website can upload garments and fetch those garments MANUALLY

In the future(this week) I want to complete the implementation of the try-on feature documentation.

https://www.loom.com/share/e6334c91d3d24bf68b97fe872724f5ff?sid=f9560fc2-6df0-4073-abf9-33ee027e52ea

**Comment by JinyueHu** [comment]
Thanks for removing the commented code and make the code clean!

File: J.Thrift/J.Thrift-api/routes/users.js [hunk 1]
```diff
@@ -89,48 +89,48 @@ router.post('/recommendations', async (req, res) => {
...
-        [sequelize.literal('(brandTrendScores[brand] || 0)'), 'DESC'], // Sort by brand trend score
-      ],
-    });
-
-    return filteredItems;
-  } catch (error) {
-    throw error;
-  }
-};
+// const getRecommendedItems = async (favoriteColors, favoriteBrands, gender) => {
```
Comment (JinyueHu) [inline J.Thrift/J.Thrift-api/routes/users.js]: Why do we comment out this file?

File: J.Thrift/J.Thrift-ui/src/components/Subbar/Subbar.jsx [hunk 2]
```diff
@@ -94,6 +98,19 @@ export default function Subbar({ handleGenderSelection, handleCategorySelection,
           </Link>
         )}
 
+        {/* {user ? (
```
Comment (JinyueHu) [inline J.Thrift/J.Thrift-ui/src/components/Subbar/Subbar.jsx]: Remember to remove it if we don't need them later.

**Comment by JinyueHu** [comment]
I think for this 
Uploaded-garment\and\fethed-garment

We probably need a separate pull request.

**Pull request merged**